pyproject.toml

```diff
@@ -7,12 +7,18 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = "^3.7"
-adbc = {git = "https://github.com/aleontiev/adbc.git", rev = "v0.2.5"}
+adbc = {path = "/home/ant/code/adbc/"} # git = "https://github.com/aleontiev/adbc.git", rev = "v0.2.6"}
 fastapi = "^0.61.1"
 huey = "^2.3.0"
 uvicorn = "^0.11.8"
+requests = "^2.24.0"
```

Comment:
Used by starlette (FastAPI dependency) but not implicitly fetched for some reason